src/context/spt_js/dg_table.js

```diff
@@ -3066,6 +3068,7 @@ spt.dg_table._search_cbk = function(evt, bvr)
         'parent_mode': parent_mode,
         'settings': settings,
         'gear_settings': gear_settings,
+        'shelf_view': shelf_view,

```

Comment:
This is allowed syntax in the EMCAScript5 specification

Section 11.1.5 in the ECMAScript 5 specification:

ObjectLiteral :
    { }
    { PropertyNameAndValueList }
    { PropertyNameAndValueList , }

I personally find that allowing it reduces an annoying error is some languages where something like